scout/adapter/mongo/case.py

```diff
@@ -321,8 +321,8 @@ def cases(
         LOG.info("Get cases with query {0}".format(query))
         if order:
             return self.case_collection.find(query)
-        else:
-            return self.case_collection.find(query).sort("updated_at", -1)
+
+        return self.case_collection.find(query).sort("updated_at", -1)
```

Comment:
I love this 🦞

Reply:
It is starting to grow on me as well. But I keep wanting to see the actual instructions executed to know which is really better. One might make less convoluted branch / jmp statements.. 😊